docs/website/data/integrations.yaml

```diff
@@ -1072,6 +1072,25 @@ integrations:
     tutorials:
       - https://hub.armo.cloud
 
+  i2scim:
+    title: i2scim.io OPA Enabled User/Group Provisioning API
```

Comment:
So this is what it looks like,

![image](https://user-images.githubusercontent.com/870638/151934526-567c4bfa-06a1-4d09-8531-4cf33403a619.png)

and 

![image](https://user-images.githubusercontent.com/870638/151934764-27391d29-3194-489a-82a3-e2c1ffa51c32.png)

🤔 the only thing that stands out to me is "OKTA", I believe they spell themselves as "Okta".

Reply:
I fixed the capitalization in the description. I also removed "OPA" from the title since the context of the ecosystem page make this overkill.  :)